cli/npm/resolvers/local.rs

```diff
@@ -323,9 +323,39 @@ fn symlink_package_dir(
 
   // need to delete the previous symlink before creating a new one
   let _ignore = fs::remove_dir_all(new_path);
+
+  #[cfg(windows)]
+  return junction_or_symlink_dir(old_path, new_path);
```

Comment:
For consistency, would it work to use an implicit return similar to the `not(windows)` path (if that doesn't work in Rust and I'm way off, sorry).

Reply:
Yeah, it doesn't compile. I'm not sure why.